lib/capistrano/configuration.rb

```diff
@@ -106,8 +106,17 @@ def timestamp
       @timestamp ||= Time.now.utc
     end
 
+    def add_filter(filter)
+      unless filter.respond_to? :filter
+        raise TypeError, "<#{filter}> is not a filter"
```

Comment:
I think a more descriptive error would be: `Custom filter does not have a public 'filter' method: #{filter}`.